src/core/common/instance.hpp

```diff
@@ -407,6 +417,14 @@ class Instance : public otInstance
     Coap::ApplicationCoap mApplicationCoap;
 #endif
 
+#if OPENTHREAD_ENABLE_APPLICATION_COAP_SECURE
+    Coap::ApplicationCoapSecure mApplicationCoapSecure;
+#endif
+
+#if OPENTHREAD_CONFIG_ENABLE_DYNAMIC_LOG_LEVEL
+    otLogLevel mLogLevel;
```

Comment:
mLogLevel is defined on line 425 and line 446...

Reply:
done.